config/constants.js

```diff
@@ -49,8 +58,8 @@ module.exports.WIN = Object.freeze({
   NETSTAT_ROUTING_TABLE: 'netstat -r',
   IPCONFIG_ALL: 'ipconfig /all',
   SWAP_USAGE: 'pagefile get AllocatedBaseSize, CurrentUsage', // this is a WMIC command. Prefix with WMIC Path
-  PING_HUB: 'ping -n 5 hub-cloud.browserstack.com',
-  PING_AUTOMATE: 'ping -n 5 automate.browserstack.com',
+  PING_HUB: 'ping -n 5 ' + this.HUB_HOST,
+  PING_AUTOMATE: 'ping -n 5 ' + this.RAILS_AUTOMATE,
```

Comment:
Can you please check in the logs if this works? `this.RAILS_AUTOMATE` includes the protocol part. And ping fails wit the protocol part.

Verify other logs as well if they are working or not with the expected result...

Reply:
similarly above...

Reply:
Fixed

Reply:
great. can be resolved.